### ISSUE TITLE
ValueError: Must specify either 2 or 3 models when using merge_alphas (not trying to merge)

### ISSUE THREAD

**Issue by haqthat** [damian0815/grate#5]
You've added some new stuff lately and now rendering fails.
I am NOT trying to merge, I am doing this:

```python
grate.py \
--prompts \
  "High detail RAW photo professional close photo of subject, on beach" \
  "subject in beautiful dress standing in a bedroom" \
  "beautiful slim subject, kneeling on tiled floor in a large bright bathroom with window during day looking off into the distance of a beautiful place battered by time and natural disasters , Intricate Detail, Elaborate Desings, HD, 8k resolution" \
  "Looking straight at the camera, stunning subject in A place flooded by the sea and with beach sand , Intricate Detail, Elaborate Desings, HD, 8k resolution" \
  "Environmental portrait of a seemly mighty subject in A place full of gardens suspended by ropes and chains" \
--negative_prompts \
  "blurry" \
  "old" \
  "ugly" \
  "cartoon" \
  "out of frame" \
--repo_ids_or_paths \
  "/workspace/EveryDream2trainer/logs/*_*/ckpts/*" \
--output_path ./grate-demo.png \
--disable_nsfw_checker
```

I get:

```python
args.merge_unet_alpha: None
5 prompts
Traceback (most recent call last):
  File "/workspace/EveryDream2trainer/grate/src/sdgrate/grate.py", line 554, in <module>
    main()
  File "/workspace/EveryDream2trainer/grate/src/sdgrate/grate.py", line 538, in main
    render_all(prompts=prompts,
  File "/workspace/EveryDream2trainer/grate/src/sdgrate/grate.py", line 322, in render_all
    raise ValueError("Must specify either 2 or 3 models when using merge_alphas")
ValueError: Must specify either 2 or 3 models when using merge_alphas
```

I added some debug and found that these params are set:
```
merge config is {'alphas': None,
 'algorithms': None,
 'unet_alphas': None,
 'text_encoder_alphas': None,
 'block_weights': None,
 'save_merge_path_prefix': None,
 'save_merge_half': True}
```

With save_merge_half defaulting to True, if following this order:
Line 254:

```python
    print(f"args.merge_unet_alpha: {args.merge_unet_alpha}")
    merge_config = {
        'alphas': args.merge_alpha,
        'algorithms': args.merge_algorithm,
        'unet_alphas': args.merge_unet_alpha,
        'text_encoder_alphas': args.merge_text_encoder_alpha,
        'block_weights': args.merge_unet_block_weights,
        'save_merge_path_prefix': args.save_merge_path_prefix,
        'save_merge_half': not args.save_merge_float32,
    }
    if all(v is None for v in merge_config.values()):
        merge_config = None
```

Line 319:

```python
    if merge_config is not None:
        if len(repo_ids_or_paths) != 2 and len(repo_ids_or_paths) != 3:
            raise ValueError("Must specify either 2 or 3 models when using merge_alphas")
```

merge_config is not None, because of the:
```python
    if all(v is None for v in merge_config.values()):
        merge_config = None
```
since save_merge_half defaults to true

Too much for me to unwind this second, not sure where you were heading with all the recent commits.
Just letting ya know.

**Comment by damian0815** [comment]
thanks, ah i knew committing to main was a bad idea 

i'll put this stuff on a branch and roll back until i've had a chance to fix it properly

**Comment by damian0815** [comment]
fixed with latest upload of sdgrate package (v0.2.1)
